pkg/standalone/run.go

```diff
@@ -148,7 +149,7 @@ func (config *RunConfig) Validate() error {
 		return err
 	}
 
-	if config.ResourcesPath != "" {
+	if len(config.ResourcesPaths) > 0 {
 		config.ComponentsPath = ""
```

Comment:
Should we add a info log here if ComponentsPath is not ""?

Reply:
If user provides both components path and resources path, it might be worth warning them again. But again, ok without it as well.